karbon/v1alpha1/karbon_registration_api.go

```diff
@@ -0,0 +1,71 @@
+package v1alpha1
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/nutanix-cloud-native/prism-go-client/internal"
+)
+
```

Comment:
We should add a compile time check for ensuring V1Alpha1 implements the interface we want. i.e.
```go
var _ K8sRegistrationService = &V1Alpha1{}
```